src/main/java/com/kramekk1/medicalClinic/MedicalClinicExceptionHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.kramekk1.medicalClinic;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class MedicalClinicExceptionHandler {
+
+    @ExceptionHandler(PatientNotFoundException.class)
+    protected ResponseEntity<String> handlePatientNotFoundException(PatientNotFoundException ex) {
```

Comment:
Zamiast zwracać po prostu message, ktory nie jest w formacie JSON to utworzmy Klase ErrorMessage i niech ta klasa Error Message zawiera pola:
message <- ten message z wyjątku
HttpStatus httpStatus <- http status z wyjatku
LocalDateTime occuredAt <- LocalDateTime.now()